build.gradle.kts

```diff
@@ -32,8 +31,6 @@ allprojects {
 }
 
 dependencies {
-    signature(libs.java8Signature) { artifact { type = "signature" } }
```

Comment:
We've already updated to Java 11 in #181 so we don't need to continue running this plugin and checking for Java 8 signatures.